docs/content/docs/osm_config_map.md

```diff
@@ -4,24 +4,141 @@ description: "OSM ConfigMap"
+| permissive_traffic_policy_mode | OpenServiceMesh.enablePermissiveTrafficPolicy | bool | true, false | `"false"` | Setting to `true`, enables allow-all mode in the mesh i.e. no traffic policy enforcement in the mesh. If set to `false`, enables deny-all traffic policy in mesh i.e. an `SMI Traffic Target` is necessary for services to communicate. |
 | prometheus_scraping | OpenServiceMesh.enablePrometheusScraping | bool | true, false | `"true"` | Enables Prometheus metrics scraping on sidecar proxies. |
-| service_cert_validity_duration | OpenServiceMesh.serviceCertValidityDuration | string | 24h, 1h30m (any time duration) | `"24h"` | Sets the service certificatevalidity duration, represented as a sequence of decimal numbers each with optional fraction and a unit suffix. |
+| service_cert_validity_duration | OpenServiceMesh.serviceCertValidityDuration | string | 24h, 1h30m (any time duration) | `"24h"` | Sets the service certificate validity duration, represented as a sequence of decimal numbers each with optional fraction and a unit suffix. |
 | tracing_enable | OpenServiceMesh.tracing.enable | bool | true, false | `"false"` | Enables Jaeger tracing for the mesh. |
 | tracing_address | OpenServiceMesh.tracing.address | string | jaeger.mesh-namespace.svc.cluster.local | `jaeger.osm-system.svc.cluster.local` | Address of the Jaeger deployment, if tracing is enabled. |
 | tracing_endpoint | OpenServiceMesh.tracing.endpoint | string | /api/v2/spans | /api/v2/spans | Endpoint for tracing data, if tracing enabled. |
 | tracing_port| OpenServiceMesh.tracing.port | int | any non-zero integer value | `"9411"` | Port on which tracing is enabled. |
 | use_https_ingress | OpenServiceMesh.useHTTPSIngress | bool | true, false | `"false"`| Enables HTTPS ingress on the mesh. |
-| outbound_ip_range_exclusion_list | OpenServiceMesh.outboundIPRangeExclusionList | string | comma separated list of IP ranges of the form a.b.c.d/x | `-`| Global list of IP address ranges to exclude from outbound traffic interception by the sidecar proxy. |
-| enable_privileged_init_container| OpenServiceMesh.enablePrivilegedInitContainer | bool | true, false | `"false"` | Enables privileged init containers for pods in mesh. When false, init containers only have NET_ADMIN |
+
+### Configure OSM ConfigMap
+### OSM Mesh Upgrade Command
+To configure values in `osm-config` use the `osm mesh upgrade` command, so that values changed in the ConfigMap are preserved. See [here](https://github.com/openservicemesh/osm/blob/main/docs/content/docs/upgrade_guide.md) for additional details on `osm mesh upgrade`.
+```bash
+# Replace osm-system with osm-controller's namespace if using a non-default namespace
+osm mesh upgrade --osm-namespace osm-system --use_https_ingress=true
+```
+> The `osm-namespace` flag can be omitted if using the default namespace.
```

Comment:
this is seeming kind of circular/repetitive since there is a comment before and note after that get at the samething. `osm-system` is the default ns, so why provide it in the example command and then tell folks to remove it? A couple options...
```suggestion
```bash
osm mesh upgrade --use_https_ingress=true [--osm-namespace <namespace>]
```
> The `osm-namespace` flag can be omitted if using the default namespace.
```

Reply:
OR 
```suggestion
```bash
# Include --osm-namespace flag if using a non-default namespace
osm mesh upgrade --use_https_ingress=true
```
```

Reply:
or some combination of the two? I'm not sure how we've shown examples of optional flags in other docs but I've seen it done the [ ] way in other repos